free-2.md

```diff
@@ -11,41 +11,62 @@ _What need is met by your idea? Who are you helping? What is that person's
+- Pixel Art Generator Tm. The need is to be able to create pixel art at varying scales using code instead of freehanding with other paint softwares 
+- I feel like current a lot of general paint softwares are built for larger scale pictures so when trying to use built in tools at that scale it can be very annoying. While this won't be as quick as freehanding on smaller indivdual elements, ideally it would be used to generate SPRITE SHEETS. So the experience if I could help would be they make one sprite then are able to copy it x times with small variations easily. 
+- In short, the need is quick and easy conversion to sprite sheets.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+- Specifically this is probably only time saving for sprite sheets (especailly environments) were slight variation is required, but one tile being a little wonkhy is fine. A dsl is appropriate because we need to be able to take small changes on a large variety of small sprites. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+- I like pixel art, and compared to the other idea I wanted to come up with something artisitic for this idea. The exiting thing is digging into pixel art holdheartedly (and likely video game and animation filters. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+- Pixel animation sprite sheet generation
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+- I think this could *ideally* be part of a bigger pixel art software (and I think some already exist) but a quick and easy drap and drop exterior coding option is where I would want to go with it. I think its the right way to interact because we want to make it part of an existing workflow, so avoiding pulling up a python document for example is much more significant then some (pregenerated) files that the initial sprite can be droped into. 
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+-Errors are infinite loops, but thats mainly it. I think when the program runs it just generates all the images and presents them as soon as they are ready. Then they can be downloaded seperately or in a sprite sheet with the original. The user requestes how many they want. 
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+- Easy: generate a bunch of sprites with small adjustments in a sheet. 
+- medium: make significant edits to each before exporting - cleaning up-- possibly also moving and placing each new sprite on the generated sheet (ie where the user wants) 
+- probably actually making the original pixel art. 
```

Comment:
I am kind of confused about what you mean by small adjustment. Is it just slightly changing the color and position of some pixels so the sprite is moving a little when animated?